packages/calendar/src/components/calendar/CalendarMonth.tsx

```diff
@@ -96,7 +98,7 @@ export function CalendarMonth<T>({
                     alignItems={"center"}
                   >
                     <StandardText color={colors.weekDayTextColor}>
-                      W
+                      {weekName}
```

Comment:
This was uppercase before, do we need to include an option for uppercase/lowercase @mattias800 @knegusen?

Reply:
Good questions. We want the translations to be generic and reusable, while not enforcing as little logic as possible.
For example, maybe some country want "Wk"? If can't force upper case in that case, while we want generic translations, so maybe the translation is "wk"?